src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -234,7 +260,7 @@ class UserService(
       _ <- directoryDAO.enableIdentity(user.id, samRequestContext).unsafeToFuture()
       _ <- registrationDAO.enableIdentity(user.id, samRequestContext).unsafeToFuture()
       _ <- cloudExtensions.onUserEnable(user, samRequestContext)
-    } yield ()
+    } yield logger.info(s"Enabled new user ${user.toUserIdInfo}")
```

Comment:
```suggestion
    } yield logger.info(s"Enabled user ${user.toUserIdInfo}")
```
Not sure if this is only called for "new" users or if we can call it to reenable a previously disabled user.